website/pages/explore/dir-content/architecture.asciidoc

```diff
@@ -1,17 +0,0 @@
-= ARCHITECTURE
-
-[.directory]
-== Architecture
-
-Architecture principles of devonfw
-
-[.common-links]
-=== Links
-
-<</website/pages/docs/master-devon4j.asciidoc_introduction.html#, devon4j architecture>>
-
-<</website/pages/docs/master-devon4ng.asciidoc_architecture.html#, devon4ng architecture>>
-
-<</website/pages/docs/master-devon4net.asciidoc_arquitecture-basics.html#, devon4net architecture>>
-
-<</website/pages/docs/master-cobigen.asciidoc_document-description.html#home.asciidoc_architecture, CobiGen architecture>>
```

Comment:
could you at least preseve parts of the links and incorporate it with the new structure? I think for example the links for all the assets are valid despite for CobiGen, where the architecture is not of interest here.